src/OrchardCore.Modules/Orchard.Users/Commands/UserCommands.cs

```diff
@@ -37,14 +33,16 @@ public class UserCommands : DefaultCommandHandler
         [OrchardSwitches("UserName,Password,Email,Roles")]
         public void CreateUser()
         {
-            var user = new User
+            var user = _userService.CreateUserAsync(
```

Comment:
Make sure you await or block on the task, otherwise you'll get a stupid race condition and the `User created successfully` message will be logged even if the task is failed.

Reply:
thanks, will do

Reply:
I will do